Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -217,6 +217,13 @@ function vtkRenderWindowInteractor(publicAPI, model) {
 
   publicAPI.bindEvents = (container) => {
     model.container = container;
+    // Clear any previous timeouts and state variables that control mouse / touchpad behavior.
+    clearTimeout(model.moveTimeoutID);
+    clearTimeout(model.wheelTimeoutID);
+    model.moveTimeoutID = 0;
+    model.wheelTimeoutID = 0;
+    wheelCoefficient = 1.0;
```

Comment:
Shouldn't you simply call unbindEvents on `model.container` ?

And move those clear timeouts in `unbindEvents` ?

Reply:
That makes sense. The `RenderWindowInteractor` assumes that it is bound to a single container at any given point. Adding an explicit unbind will make it more obvious, and the right thing to do.

Reply:
@finetjul  updated the branch

Reply:
I also think that the (un)binding should be done by `setContainer()`

Reply:
One thought regarding `setContainer`: we should be able to add it without breaking bind/unbindEvents. Just need to be sure that setContainer operates similarly to bind/unbindEvents, and that if someone calls setContainer and then bindEvents, we don't get two listeners registered.

Reply:
What behavior do we want for `setContainer(null)` ?

Reply:
> What behavior do we want for setContainer(null) ?

`setContainer(null)` it should simply unbind the current container. No binding.

> `we should be able to add it without breaking bind/unbindEvents`. 

@floryst are you saying we should still support (backward compatibility) the case were users do not call `setContainer()` but only bind/unbind ?

Reply:
That's my preference, yes. Current applications use bind/unbind, so setContainer should exist as an alternative. We should document a preference for setContainer in bind/unbind docs. We could mark bind/unbind as deprecated and remove them in the future.

Reply:
Pushed another update based on both of your suggestions.